<?xml version="1.0" encoding="UTF-8"?><testrun name="TestAlgoritmos" project="PracticaAlgoritmos" tests="3" started="3" failures="1" errors="0" ignored="0">
  <testsuite name="tests.TestAlgoritmos" time="0.02" displayname="TestAlgoritmos" uniqueid="[engine:junit-jupiter]/[class:tests.TestAlgoritmos]">
    <testcase name="testesPrimo" classname="tests.TestAlgoritmos" time="0.018" displayname="testesPrimo()" uniqueid="[engine:junit-jupiter]/[class:tests.TestAlgoritmos]/[method:testesPrimo()]">
      <failure>
        <expected>true</expected>
        <actual>5040</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;true&gt; but was: &lt;5040&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1145)&#13;
	at tests.TestAlgoritmos.testesPrimo(TestAlgoritmos.java:20)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;

      </failure>
    </testcase>
    <testcase name="testfactorial" classname="tests.TestAlgoritmos" time="0.001" displayname="testfactorial()" uniqueid="[engine:junit-jupiter]/[class:tests.TestAlgoritmos]/[method:testfactorial()]"/>
    <testcase name="testfibonacci" classname="tests.TestAlgoritmos" time="0.001" displayname="testfibonacci()" uniqueid="[engine:junit-jupiter]/[class:tests.TestAlgoritmos]/[method:testfibonacci()]"/>
  </testsuite>
</testrun>
